content/docs/guides/node-exporter.md

```diff
@@ -15,7 +15,7 @@ NOTE: While the Prometheus Node Exporter is for *nix systems, there is a [WMI ex
 
 ## Installing and running the Node Exporter
 
-The Prometheus Node Exporter is a single static binary that you can install [via tarball](#tarball-installation). Once you've downloaded it from the Prometheus [downloads page](/download#node_exporter) extract it, and run it:
+The Prometheus Node Exporter is a single static binary that you can install via tarball. Once you've downloaded it from the Prometheus [downloads page](/download#node_exporter) extract it, and run it:
```

Comment:
It seems like this is an unrelated change that made it into this PR?

Reply:
It's an unrelated change but I felt it was small enough to include here (it fixes a broken link). I can put it in a separate PR if you'd like.